modules/photons_control/clean.py

```diff
@@ -0,0 +1,188 @@
+from photons_control.script import FromGenerator
+
+from photons_app.tasks import task_register as task
+from photons_app.errors import PhotonsAppError
+
+from photons_messages import LightMessages, DeviceMessages
+
+
+def CleanConfig(indication=None, duration=None, **kwargs):
```

Comment:
```suggestion
def SetCleanConfig(indication=None, duration=None, **kwargs):
```

It should be named as a verb given it changes things on the device

Reply:
Another good reason for this draft PR. 😄